template/src/tests/__mocks__/fileMocks.js

```diff
@@ -0,0 +1,3 @@
+// This fixed an error related to the CSS and loading gif breaking my Jest test
+// See https://facebook.github.io/jest/docs/en/webpack.html#handling-static-assets 
+module.exports = 'test-file-stub';
```

Comment:
I wonder if these could just use `identity-object-proxy` like the CSS is?

Reply:
Good question I will try that the next days / check again the identity-object-proxy.

Reply:
I've read a bit through Jest documentation and about `moduleNameMapper`and it seems that for static file mocks, we are free to use what we want jest just replaces all references with `test-file-stub`. Therefore using ìdentity-object-proxy` would also work, at least my current testing is working but I don't have much of files to mock currently.